tests/test_e2e.py

```diff
@@ -115,7 +111,7 @@ def test_tensorrtllm_e2e(self, protocol, setup_and_teardown):
         utils.wait_for_server_ready()
 
         self._infer(model, prompt=PROMPT, protocol=protocol)
-        self._profile(model, backend="tensorrtllm", protocol=protocol)
+        self._profile(model, backend="tensorrtllm")
```

Comment:
Just a question for my own future knowledge, these changes don't need to be added in this PR:
- is there anything interesting about genai-perf we can also test for here? 
- ex: parsing an output CSV file for expected values or something? 
- ex: testing both streaming and non-streaming
- ...

Reply:
We could look to ensure there is some form of output (e.g. the expected chart name and/or some of the metrics). The tool is still in the alpha phase with a lot of changes, so those tests here could end up breaking as things change there.

I think the zero exit code should be sufficient to know that it ran successfully though.